src/frontend/src/Components/shared/Filters.scss

```diff
@@ -41,6 +41,42 @@ button.btn.open-filters {
   .filters-title {
     margin: 1rem 1rem 0;
     font-size: 1.125rem;
+    margin-right: 0rem;
+    &__container {
```

Comment:
please update the following:
- remove gap
- remove justify-content
- add margin: 12px 12px 0;

Reply:
Hi James, I'm still working on the styling, need some other updates.